transport/local/transport.go

```diff
@@ -0,0 +1,64 @@
+package local
+
+import (
+	"io"
+	"os"
+	"sync"
+
+	"github.com/kcarretto/paragon/transport"
+)
+
+type Transport struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported type Transport should have comment or be unexported